Description: Modify BASECONFIG_DIR
 Modify the BASECONFIG_DIR default to find the baseconfig.txt
 and baseconfig_sw.txt from /usr/share/doc/gns3/
Author: Daniel Lintott <daniel@serverb.co.uk>
Forwarded: Not needed, debian specific
---
--- a/src/GNS3/Config/Defaults.py
+++ b/src/GNS3/Config/Defaults.py
@@ -48,7 +48,7 @@
 elif sys.platform.startswith('win'):
     BASECONFIG_DIR = ''
 else:
-    BASECONFIG_DIR = '/usr/local/share/examples/gns3/'
+    BASECONFIG_DIR = '/usr/share/gns3/'
 
 # Default path to qemuwrapper
 if sys.platform.startswith('win'):
